From c4d4c656b38d9cbe021664e6d0aa47e432b5e4cd Mon Sep 17 00:00:00 2001
From: Dominik Loidolt <dominik.loidolt@univie.ac.at>
Date: Wed, 22 Jan 2025 14:39:56 +0100
Subject: [PATCH 1/2] fix: update meson options to avoid deprecation warnings

- Remove quotes from boolean values in meson_options.txt
- see https://mesonbuild.com/Release-notes-for-1-1-0.html#coercing-values-in-the-option-function-is-deprecated
---
 meson_options.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/meson_options.txt b/meson_options.txt
index 8e66784..c081b59 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1,3 +1,3 @@
-option('extension_fixture', type: 'boolean', value: 'false', description: 'Whether to enable the fixture extension.')
-option('extension_memory', type: 'boolean', value: 'false', description: 'Whether to enable the memory extension.')
-option('support_double', type: 'boolean', value: 'false', description: 'Whether to enable double precision floating point assertions.')
+option('extension_fixture', type: 'boolean', value: false, description: 'Whether to enable the fixture extension.')
+option('extension_memory', type: 'boolean', value: false, description: 'Whether to enable the memory extension.')
+option('support_double', type: 'boolean', value: false, description: 'Whether to enable double precision floating point assertions.')
--
2.49.0
